docs/DesignDocs/RemoteAttestationCollaterals.md

```diff
@@ -473,6 +473,8 @@ Current set of claims definitions:
 | tcb_date                   | oe_datetime_t      | Remote Only   |The date and time when the evidence's TCB level was certified.       |
 | validity_from              | oe_datetime_t      | Remote Only   |Overall datetime from which the evidence and endorsements are valid. |
 | validity_until             | oe_datetime_t      | Remote Only   |Overall datetime at which the evidence and endorsements expire.      |
+| ue_id                      | uint8_t[16]        | Remote Only   |Universal Entity Identity (qe_id for SGX).                           |
+| hardware_model             | uint8_t[6]         | Remote Only   |Hardware family/ model identity (fmspc for SGX).                     |
```

Comment:
My reading of https://datatracker.ietf.org/doc/draft-ietf-rats-eat/ section 3.7 is that hardware_model should only be used when oemid is also present.   Can oemid be derived from other fields here?  I think yes, just want to verify.

Reply:
About oemid, I have created an issue- https://github.com/openenclave/openenclave/issues/4476. Will address this as a separate PR.